### DISCUSSION TITLE
Output doesn't have any style

### DISCUSSION THREAD

**Discussion by atenadadkhah** [awcodes/filament-tiptap-editor#308] [Q&A]
How can I style the output cause in the editor, the elements have beautiful style but when I render that in a page, most of them are gone

**Comment by awcodes** [comment]
You have to style that yourself. I can't presume to know anything about how your site needs to be styled. You could use the plugins styles sheet as a guide and start there. I usually just use tailwind typography plugin for my frontend with some styles for grids etc. 

It has to be separate stylesheets though because things need to have different styling in the editor.

**Comment by scottjo** [comment]
Sort of following on from this. Just installed the field editor and I think its working as expected however, when I add two columns in the editor and fill them in the output is 'unstyled' including the columns - just a single column is rendered.  I would have expected the layout to be output even if the style hasn't been.

Any advice?

Thanks in advance

Jon

**Reply by awcodes** [reply]
If you aren't seeing two columns in the editor then you didn't setup a custom theme correctly according to the Readme.

If you you aren't seeing two columns on output, it's because you have to style it with your own css file.

**Reply by scottjo** [reply]
Thanks for your feedback - I am seeing them ok in the editor. I'll take a butchers at the rendered output and see what I can latch onto. Thanks again

**Reply by alaa007** [reply]
Im using bootstrap in the frontend, what kind of styling do I need to add to make the grid system work fine? Can anyone help?
